t/plugin/example.t

```diff
@@ -339,3 +339,56 @@ plugin body_filter phase, eof: false
 plugin delayed_body_filter phase, eof: false
 plugin body_filter phase, eof: true
 plugin delayed_body_filter phase, eof: true
+
+
+
+=== TEST 14: lua body filter
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1',
+                ngx.HTTP_PUT,
+                [[{
+                    "uri": "/anything",
+                    "plugins": {
+                        "example-plugin": {
+                            "i": 0,
+                            "lua_proxy_upstream": true,
+                            "request_uri": "http://httpbin.org/get",
```

Comment:
rename this `request_uri`  to indicate it's used to change the rseponse body, it make me confused with `upstream.nodes`